### PR TITLE
feat(icons): add PaletteIcon

### PR DESCRIPTION
**Pull request by nick-lanners**
## Jira issue
[EC-2263 - Front end: Update store switcher to be visible on settings sidebar](https://myzonos.atlassian.net/browse/EC-2263)

## Description

This PR creates the PaletteIcon which is needed for the new Settings Sidebar in Dashboard

## Todo

- [x] Bump version and add tag

### CODE REVIEW

File: src/icons/IconIndex.ts [hunk 1]
```diff
@@ -192,6 +192,7 @@ export { MonitorDuotoneIcon } from './MonitorDuotoneIcon';
 export { MonitorIcon } from './MonitorIcon';
 export { NightDuotoneIcon } from './NightDuotoneIcon';
 export { NightIcon } from './NightIcon';
+export { PaletteIcon } from './PaletteIcon';
```
Comment (ckgardner) [inline src/icons/IconIndex.ts:195]: I think you need to add the duotone icon here too. 😁

Comment (nick-lanners) [reply src/icons/IconIndex.ts:196]: @ckgardner Oh man, I'm a mess. Done it.

**Comment by ckgardner** [comment]
> > Is there a duotone for this?
> 
> @ckgardner Whoops, yes there is. Just added it.

Theres some instructions on adding icons in Amino in the readme. You should have some svg files added in this PR as well and then run the `svgs:react:icons` command. It will auto generate everything for you. Let me know, and I can pop over to help.

**Review by ckgardner — DISMISSED** [review]
This looks more like it! Thanks for doing this, I need this icon for a task I'm working on as well!

**Review by ckgardner — APPROVED** [review]

**Pull request merged**